demos/demos_by_use_case/bio/MetagenomicDemo.ipynb

```diff
@@ -0,0 +1,1975 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Accelerating metagenomic analysis with [Graphistry](graphistry.com) focusing on viral tracing over time\n",
+        "\n",
+        "## [viral calling pipeline here](https://github.com/dcolinmorgan/viral_snake)"
+      ],
+      "metadata": {
+        "id": "ozrca88hza85"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Using GPU-accelerated UMAP + DBScan analysis & visualization, metagenomic samples' bacterial compositions can be clustered and compared faster and much more easily explored.\n",
+        "\n",
+        "*   Task: Analyze metagenomic samples for similarity\n",
+        "*   Data: time series samples\n",
+        "**   563 samples collected from 84 donors, producing 4 dense long-term time series (up to 1 sample every other day during 18 months)\n",
```

Comment:
`**` => `*`

Reply:
Clarify what a node would be,  or what is getting clustered

E.g.

* Nodes: Clustering the species component extracted from time-stamped patient samples, e.g., tuple of `<idx=0, time=1, donor=aa, species=abc>`, each node is `<idx=0, rel_time=0, value=abc>`, and clustering is on species value text similarity